modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -37,8 +37,30 @@ function inope_policy_review_menu() {
  *   policy nodes.
  */
 function inope_policy_review_tab_title_callback($node) {
+
+  global $user;
+  $account = $user;
+
   // Create the string to be used as the tab title.
   $title = t('Review of @node_title', array('@node_title' => $node->title));
+
+  // Retreive all notes related to this node, account, and vid_timestamp.
+  $results = inope_policy_review_select_query_all($node, $account);
+  $node_revision = inope_policy_review_select_query_timestamp($node);
+
+  // Change the title based off last submission.
+  if (empty($results)) {
+    $title .= ' (NEW)';
+    drupal_set_message(t('This is a new policy, please provide feedback.'), 'status', $repeat = FALSE);
```

Comment:
When a user visits the node page a drupal_set_message posts a message requesting the user to provide feedback and the Review of @node_title page title is changed.  The idea behind this was it is asking the user to "Click Me, I have changes"

Would you have actually changed the node view title?